toolkit/pythonit_toolkit/pastaporto/tokens.py

```diff
@@ -24,3 +24,15 @@ def decode_service_to_service_token(
         algorithms=["HS256"],
         options={"require": ["exp", "iss", "aud", "iat"]},
     )
+
+
+def generate_token(secret, issuer: str, audience: str, expires_in: str = "1m"):
```

Comment:
can we add some unit-tests for this?

Reply:
we should have a more specific name for this function

also expiration should always be 1 minute so let's not pass it as parameter

```suggestion
def generate_service_to_service_token(secret, issuer: str, audience: str):
```

Reply:
also type hint missing in secret :)

```suggestion
def generate_token(secret: str, issuer: str, audience: str, expires_in: str = "1m"):
```

can we also raise a ValueError `if not secret`? same for issuer and audience, those should always be there and not be empty